### DISCUSSION TITLE
Next.js SVG imports as components

### DISCUSSION THREAD

**Discussion by ScottAgirs** [vercel/next.js#29293] [Ideas]
Hello folks,

CRA has this neat way of importing `.svg` files as components via
```js
  import { ReactComponent as MyLogo } from '../icons/ijs.svg';

  const Header = () => (
    <>
      ...
      <MyLogo />
    </>
  )
```

Any chance there's something on the horizon for this within Next.js? 🤞🏼

**Comment by wwwdepot** [comment]
See example https://github.com/vercel/next.js/tree/canary/examples/svg-components

**Reply by ScottAgirs** [reply]
Wooohoohooo, haha, totally missed this, thanks!! 🙌🏼
